src/components/Tag/Tag.scss

```diff
@@ -0,0 +1,32 @@
+.Tag {
+    display: flex;
+    align-items: center;
+
+    padding: var(--spacing-normal) var(--spacing-2x);
+    border-radius: var(--border-radius);
+
```

Comment:
when onClick is defined, cursor should be a pointer.

Reply:
Implemented.